String/anagramshash.cpp

```diff
@@ -0,0 +1,56 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+	int arr[256];
+		 for(int i = 0; i <256;i ++)
+	 {
+	 	arr[i]=0;
+	 }
+	 	string str;
+	string str1;
+	 cout<<"enter string1"<<endl;
+	 cin>>str;
+	 cout<<"enter string2"<<endl;
+	 cin>>str1;
+
+	if(str.size()==str1.size())
```

Comment:
Given the extra lines of "enter string1", "enter string2", this doesn't look like an input question. So, remove this as well. If it's an assignment question, let me know.